{"uuid":"ec4ad67d-6c4c-42f5-8bf9-11f8c84dffba","historyId":"7918f492f46d0b919ebcb06a593cdd75","fullName":"rms.testcases.rmsTestCase.rmsFirstTestCase","labels":[{"name":"package","value":"rms.testcases.rmsTestCase"},{"name":"testClass","value":"rms.testcases.rmsTestCase"},{"name":"testMethod","value":"rmsFirstTestCase"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Default XmlTest name 3e71f814-747a-4c64-bb68-0ad73a62d554"},{"name":"subSuite","value":"rms.testcases.rmsTestCase"},{"name":"host","value":"Sunil-Chauhan"},{"name":"thread","value":"21388@Sunil-Chauhan.TestNG-test=Default XmlTest name 3e71f814-747a-4c64-bb68-0ad73a62d554-1(22)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"rmsFirstTestCase","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Exception occured while opening RMS excel workbook \\test-data\\RMS_Data\\RMSExcel.xlsx","trace":"java.lang.RuntimeException: Exception occured while opening RMS excel workbook \\test-data\\RMS_Data\\RMSExcel.xlsx\r\n\tat rms.testData.excel.RMSExcelUtils.readRMSExcelSheets(RMSExcelUtils.java:43)\r\n\tat rms.testcases.rmsTestCase.rmsFirstTestCase(rmsTestCase.java:22)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:577)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:258)\r\n\tat framework.testNG.TestMonitor.run(TestMonitor.java:55)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:272)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:656)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)\r\n\tat java.base/java.lang.Thread.run(Thread.java:833)\r\nCaused by: java.lang.RuntimeException: The specified excel not found in the path \\test-data\\RMS_Data\\RMSExcel.xlsx\r\n\tat framework.utils.ExcelUtils.openExcelWorkBook(ExcelUtils.java:33)\r\n\tat rms.testData.excel.RMSExcelUtils.readRMSExcelSheets(RMSExcelUtils.java:41)\r\n\t... 17 more\r\nCaused by: java.io.FileNotFoundException: \\test-data\\RMS_Data\\RMSExcel.xlsx (The system cannot find the path specified)\r\n\tat java.base/java.io.FileInputStream.open0(Native Method)\r\n\tat java.base/java.io.FileInputStream.open(FileInputStream.java:216)\r\n\tat java.base/java.io.FileInputStream.<init>(FileInputStream.java:157)\r\n\tat framework.utils.ExcelUtils.openExcelWorkBook(ExcelUtils.java:31)\r\n\t... 18 more\r\n"},"stage":"finished","steps":[{"name":"<a href=../screenshot/rmsFirstTestCase_2023-May-29-16-55-48.PNG target=_blank>SCREENSHOT</a>","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1685359548502,"stop":1685359548503}],"attachments":[],"parameters":[],"start":1685359548484,"stop":1685359548511}